src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -2044,13 +2044,20 @@ public void test_hasSubmodules() throws Exception {
         assertFalse(w.git.hasGitModules());
     }
 
+    private boolean isJava6() {
+        if (System.getProperty("java.version").startsWith("1.6")) {
```

Comment:
Yes, as is 1.6.0_0 and 1.6.0_14 and 1.6.0_11 and 1.6.0_15 and 1.6.0_05 and 1.6.0_45.  The question is trying to answer "is this Java 6 (and not Java 7 or Java 8)", not "is this a specific version of Java 6".

Do you see a problem with that technique?

Is there a better technique for deciding that the currently executing Java runtime environment does not have support for symbolic links?